INSTALL.md

```diff
@@ -49,21 +49,11 @@ Prerequisites
 
         sudo apt-get install git-core
 
-5.  Memcached
-
-    By default the MusicBrainz server requires a Memcached server running on the
-    same server with default settings. To install Memcached, run the following:
-
-        sudo apt-get install memcached
-
-    You can change the memcached server name and port, or configure other datastores
-    in lib/DBDefs.pm.
-
 6.  Redis
```

Comment:
This should be renumbered to 5 (and the following entries as well). For some reason, GitHubs rendered versions number these correctly, but https://raw.githubusercontent.com/mwiencek/musicbrainz-server/redis/INSTALL.md looks incorrect.